system-addon/bootstrap.js

```diff
@@ -150,6 +150,12 @@ function onBrowserReady() {
       Services.prefs.setIntPref("browser.newtabpage.activity-stream.topSitesCount", rows * 6);
```

Comment:
We should migrate the old `newtabpage.rows` pref directly to `topSiteRows`.

Reply:
(Yes, I realize there's a conflict if both prefs need to be migrated, but oh well. :p)